R2API/ItemAPI.cs

```diff
@@ -342,9 +344,27 @@ public class CustomItem {
             ItemDef.pickupModelPrefab = pickupModelPrefab;
             ItemDef.pickupToken = pickupToken;
             ItemDef.tags = tags;
-            ItemDef.tier = tier;
             ItemDef.unlockableDef = unlockableDef;
 
+            //If the tier isnt assigned at runtime, load tier from addressables, this should make it so mods that add items dont break.
+            //We dont want to set the .tier directly, because that'll attempt to load the itemTierDef via the itemTierCatalog, and we cant
+            //Guarantee said catalog has been initialized at that point
+            if (tier != ItemTier.AssignedAtRuntime) {
+                ItemDef._itemTierDef = LoadTierFromAddress(tier);
+                return;
```

Comment:
is it intentional that this `return` statement causes `ItemDisplayRules` to never be set?

perhaps we should avoid using `return` as control flow in the ctor

Reply:
I swear i placed the tier logic after the item display rule set for the item was created, that's on me